hostmgr/cluster.go

```diff
@@ -110,56 +89,28 @@ func NewCluster(baseDir string, gitStore bool, logger micrologger.Logger) (*Clus
 // CreateNewHost creates a new host with the given serial.
 func (c *Cluster) CreateNewHost(serial string) (*Host, error) {
 	serial = strings.ToLower(serial)
-	hostDir := path.Join(c.baseDir, strings.ToLower(serial))
+	hostDir := path.Join(c.baseDir, serial)
```

Comment:
strings.ToLower was used multiple times on the same string that was already lowered